core/vm/actor/builtin/v3/multisig/multisig_actor.hpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/actor_method.hpp"
+#include "vm/actor/builtin/v0/multisig/multisig_utils.hpp"
+#include "vm/actor/builtin/v2/multisig/multisig_actor.hpp"
+#include "vm/actor/builtin/v3/multisig/multisig_actor_state.hpp"
+
+namespace fc::vm::actor::builtin::v3::multisig {
+  using primitives::ChainEpoch;
+  using primitives::EpochDuration;
+  using utils::multisig::MultisigUtils;
+
+  struct Construct : ActorMethodBase<1> {
+    using Params = v2::multisig::Construct::Params;
+
+    ACTOR_METHOD_DECL();
+    static outcome::result<Result> execute(Runtime &runtime,
+                                           const Params &params,
+                                           const MultisigUtils &utils);
+  };
+  CBOR_TUPLE(
+      Construct::Params, signers, threshold, unlock_duration, start_epoch)
```

Comment:
`utils` were not used in `Construct`, so it is identical to v2::Construct, so
```suggestion
  using Construct = v2::multisig::Construct;
```

Reply:
done